app/src/main/java/com/eveningoutpost/dexdrip/utils/LibreTrendUtil.java

```diff
@@ -165,6 +198,45 @@ boolean IsTimeValid(long timeId) {
         }
         return m_points;
     }
+
+    public void AddLibreblock(LibreBlock libreBlock) {
+        if(!libreBlock.reference.equals(m_libreTrendLatest.SensorSN)) {
+            Log.i(TAG, "Detected a sensor change (or a new one) new sn is " + libreBlock.reference);
```

Comment:
"Detected a sensor change (or a new one); new serial number is: "

Reply:
Done.